.github/workflows/pages.yml

```
@@ -30,4 +30,5 @@ jobs:
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          external_repository: mit-dci/opencbdc-tx-pages
           publish_dir: ./doxygen_generated/html
```

```suggestion
        uses: peaceiris/actions-gh-pages@v4
        with:
          deploy_key: ${{ secrets.TX_PAGES_DEPLOY_KEY }}
          external_repository: mit-dci/opencbdc-tx-pages
          publish_dir: ./doxygen_generated/html
```

I believe this change is sufficient to enable publishing to the default branch of the docs repository (largely copied from my docs MVP test above, tweaked appropriately). @maurermi if it makes sense, I can add the new secret and variable name changes to my test repo and deploy from that repo first.

---

Yes that sounds good. I can commit this change as well